UNRELEASED.md

```diff
@@ -51,3 +51,6 @@ See [`CHANGELOG.md`](CHANGELOG.md) for real-world examples of good changelog ent
 
 - react-native-bpk-component-horcrux:
   - Fixed issue where `BpkHorcrux` would occasionally possess the living.
+
```

Comment:
Looks good now, but this should be at the top of the file, this part is just examples.